Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -1381,6 +1381,20 @@ theorem lt_ord_succ_card (o : Ordinal) : o < (succ o.card).ord :=
   lt_ord.2 <| lt_succ _
 #align cardinal.lt_ord_succ_card Cardinal.lt_ord_succ_card
 
+theorem card_le_iff {o : Ordinal} {c : Cardinal} : o.card ≤ c ↔ o < (succ c).ord := by
+  rw [lt_ord, lt_succ_iff]
+
+/--
+A variation on `Cardinal.lt_ord` using `≤`: If `o` is no greater than the
+initial ordinal of cardinality `c`, then the cardinal of any of its realizations
+is no greater than `c`.
+
+The converse, however, is false (for instance, `o = ω+1` and `c = ℵ₀`).
+-/
+lemma card_le_of_le_ord {o : Ordinal} {c : Cardinal} (ho : o ≤ c.ord) :
+    o.card ≤ c := by
+  rw [← card_ord c]; exact Ordinal.card_le_card ho
```

Comment:
You can do this but it's longer and depends on `ord_lt_ord` which appears later, so forget about it.
```suggestion
    o.card ≤ c :=
  card_le_iff.2 (ho.trans_lt <| ord_lt_ord.2 <| lt_succ c)
```